### ISSUE TITLE
Ads Not Displayed On Android Oreo Devices

### ISSUE THREAD

**Issue by syedabuthahirm** [sbugert/react-native-admob#298]
Version: 2.0.0-beta.3

Ads aren't showing on my oreo device but ads are working fine in Earlier Versions.

**Comment by ereenyessamhanna** [comment]
It aslo works for me on the earlier devices before oreo . is there any news about this issue ?

**Comment by syedabuthahirm** [comment]
nope. I didn't resolve that issue. i tried, but i skipped that part. you can try v1.3.2.

**Comment by anhhtz** [comment]
Same problem on Lenovo S660 (Android 4.4.2)

**Comment by re-LIFE** [comment]
Same problem ads not show in oreo

**Comment by julianov** [comment]
Hello All. Im having the same problems. Is there any news?

**Comment by julianov** [comment]
Hello All. Im having the same problem. Is there any news?

**Comment by yalcinozer** [comment]
I solved issue by adding the line below to my config.xml file. For those who still search for a solution
<preference name="android-targetSdkVersion" value="26" />
